app/views/pseuds/_pseuds_form.html.erb

```diff
@@ -62,8 +62,6 @@
     <dt><%= f.label :ticket_number, class: "required" %></dt>
     <dd>
       <%= f.text_field :ticket_number, class: "required" %>
-      <%= live_validation_for_field("pseud_ticket_number", numericality: true) %>
-      <p class="footnote"><%= t(".ticket_footnote") %></p>
```

Comment:
To solve the test failure, the locale string associated with this key has to be removed from `config/locales/views/en.yml`. Same for the other removed `t()` call.